### DISCUSSION TITLE
large file issue

### DISCUSSION THREAD

**Discussion by gargi806** [community/community#108940] [New to GitHub]
remote: Resolving deltas: 100% (1445/1445), done.
remote: error: Trace: a305bc1024df0b9030b5a4ff469db9990120bc9d366f13c72e2a8084f52ba4d9
remote: error: See https://gh.io/lfs for more information.
remote: error: File .parcel-cache/data.mdb is 158.25 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: File .parcel-cache/data.mdb is 195.25 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: File .parcel-cache/data.mdb is 159.75 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.

this is the error when i am trying to push my code on git

**Comment by Ahsankhalid618** [comment]
Remove the .parcel-cache directory from Git: If the .parcel-cache directory contains files that don't need to be version-controlled (which is often the case with build caches), you can remove it from your repository and prevent it from being tracked: 
Use the following command to remove the directory from your repository:
 `git rm -r --cached .parcel-cache`
To prevent the directory from being tracked in the future, add it to your .gitignore file: `echo ".parcel-cache" >> .gitignore` 
Commit and Push: After removing the directory and updating your .gitignore file, commit your changes and push again.
